crates/ruff_python_formatter/tests/snapshots/format@expression__fstring.py.snap

```diff
@@ -662,7 +712,7 @@ hello {
 # Implicit concatenated f-string containing quotes
 _ = (
     "This string should change its quotes to double quotes"
-    f'This string uses double quotes in an expression {"woah"}'
+    f"This string uses double quotes in an expression {"it's a quote"}"
     f"This f-string does not use any quotes."
 )
```

Comment:
Is this correct? The outer single quotes got formatted to use double quotes even though the expression has a double quoted string, thus not using alternate quote style.

Reply:
Yes. The idea is that we use the same quote selection criteria as everywhere else and we keep using double quotes to avoid unnecessary escaping the single quote